### PR TITLE
DOC/projections: Make projection name consistent

### PR DESCRIPTION
**Pull request by yvonnefroehlich**
**Description of proposed changes**

Related to the points 3. and 4. of https://github.com/GenericMappingTools/pygmt/issues/3405#issue-2476920122.

Currently the projection names are adjusted based on the name used in the projection table.

Related PyGMT PR at https://github.com/GenericMappingTools/pygmt/pull/3407.

**Reminders**

- [ ] Make sure that your code follows our style. Use the other functions/files as a basis.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Describe changes to function behavior and arguments in a comment below the function declaration.
- [ ] If adding new functionality, add a detailed description to the documentation and/or an example.

### CODE REVIEW

**Comment by joa-quim** [comment]
Hmm, lots of changes here. While I agree on dropping the word `projection` for most (all?) cases, I cannot agree with all proposed changes. For example `Mercator projection` to `Mercator cylindrical` (all Mercators are cylindrical), and a few others. Maybe do this in two phases, leaving out on the first step the projection renamings.

**Review by joa-quim — APPROVED** [review]
@WalterHFSmith Walter, do you have opinion on dropping "equal-area" in some projection names?

**Comment by WalterHFSmith** [comment]
Dear All,
I looked at the Wikipedia page but I am not sure what you are proposing to change. 
I think it is perhaps unfortunate or not helpful that the Wikipedia page starts by giving each projection a name, then lists things like properties and types separately. Also that the names are not unique in the sense that one has Mercator, Web Mercator (and in GMT at least) Transverse Mercator and Oblique Mercator.
Mercator is an example of a problem, for parsing names, I suppose, in that "Mercator" without other modifying words could be assumed to mean a conformal and cylindrical development with the cylinder tangent to the Equator, and furthermore, on the Earth, using a development that is conformal for the Ellipsoid rather than a sphere. If we take the idea of being cylindrical and conformal and wrap the paper around another way, then we can have these true properties in Transverse and Oblique aspect but true only for a sphere and not for an ellipsoid.
Equal-Area (Authalic is a synonym for Equal-Area) is a property, and it does not imply a projection type; there can be Azimuthal, Conic, Cylindrical projections which are Equal-Area.
The projection type does not imply having, or not, the property of being Equal-Area. For example, there are both Conformal and Equal-Area conic and cylindrical and azimuthal projections.
Equal-Area cannot be Conformal, and Conformal cannot be Equal-Area, and so these terms imply exclusion of the other. But they don't uniquely name a projection.
It is also possible to develop the sphere or ellipsoid so that the map shows true distance and azimuth from one point at the map center.
Some of these are named for people and some aren't. And among those that are, such as Mercator, their generalization to other aspects isn't always clearly standardized. [Indeed, the problem of a transverse cylindrical that went from Canada across the North Pole to Russia is what originally lead me to put Auxiliary Latitudes into GMT.]


So what is the problem we are trying to solve? A standardization of nomenclature?

**Comment by joa-quim** [comment]
Thanks Walter. I won't answer what the problem tried to be solved because I'm not sure either. And from what you wrote I deduce that we should not drop the Equal-Area" from names and that instead it should be understood as that the implementation is using Authalic sphere.

In order to see the proposed changes click on the "Files changed" at the top of this page

image

**Comment by yvonnefroehlich** [comment]
Thanks @WalterHFSmith for your very detailed comment!

The idea of this PR is to make the projection name consistent between the projection table (https://docs.generic-mapping-tools.org/dev/reference/map-projections.html) and the projection explanations.
The starting point was PyGMT (https://github.com/GenericMappingTools/pygmt/issues/3405, https://github.com/GenericMappingTools/pygmt/pull/3407). We prefer to have the same projection names in the projection table (https://www.pygmt.org/dev/techref/projections.html) and for the projection explanations (https://www.pygmt.org/dev/projections/index.html). Additionally to this, I feel it would be good to make the projection names consistent between the GMT and PyGMT documentation. Thus, I started working on this upstream PR and plan to apply these changes in the same way to the PyGMT documentation. 

For example,
- in the table it is "Eckert IV equal area" and "Eckert VI equal area" but the heading later is ["Eckert IV and VI projection"](https://docs.generic-mapping-tools.org/dev/reference/map-projections.html#jk)
- table "Mercator cylindrical" but heading ["Mercator"](https://docs.generic-mapping-tools.org/dev/reference/map-projections.html#jm)
- table "Equidistant cylindrical" but heading [" Cylindrical equidistant projection"](https://docs.generic-mapping-tools.org/dev/reference/map-projections.html#jm)

**Comment by joa-quim** [comment]
OK, so things are now clear on my mind (mean, opinion). 

We should **not** drop the "equal area" from names in the table as it provides important information. The projection description doesn't have it in the header but has in the description.

We could drop the ``cylindrical``  from Mercator but that we have a `Equidistant cylindrical` and a `Equidistant conic`. So leave the Mercator as it is now.

Finally the `Equidistant cylindrical` _vs_ heading ` Cylindrical equidistant projection` case. Here the header should be changed to `Equidistant cylindrical projection.`

Even the `projection` word should should stay in the headers. I don't see now any reason why the table (which a summary) and the headers should be exactly equal.

**Comment by WalterHFSmith** [comment]
Yvonne and Joaquim,
I am all for standardizing names and making things clear and avoiding confusion.

As far as I know, the only projection known as Mercator is a cylindrical projection, so perhaps one can drop the term "cylindrical" from Mercator, if that helps to standardize the nomenclature.

But note that there are many cylindrical projections (conformal = Mercator, but also equal area and others) and they can be in various aspects (cylinder tangent on any great circle, not necessarily the Equator), so in general we cannot drop "cylindrical" from a name.

Sorry, these comments from me are not much help.

I am glad you are doing this.

Walter

File: doc/rst/source/reference/map-projections.rst [hunk 1]
```diff
@@ -1358,8 +1358,8 @@ distributions like hydrocarbon and mineral resources, etc.
 
 .. _-Jv:
 
-Van der Grinten projection (**-Jv** **-JV**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Van der Grinten (**-Jv** **-JV**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:1362]:
```suggestion
Van der Grinten projection (**-Jv** **-JV**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/rst/source/reference/map-projections.rst [hunk 2]
```diff
@@ -1305,8 +1305,8 @@ The same script, with **s** instead of **f**, yields the Eckert VI map:
 
 .. _-Ji:
 
-Sinusoidal projection (**-Ji** **-JI**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Sinusoidal (**-Ji** **-JI**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:1309]:
```suggestion
Sinusoidal projection (**-Ji** **-JI**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/rst/source/reference/map-projections.rst [hunk 3]
```diff
@@ -1254,8 +1254,8 @@ Again centered on Greenwich, the example below was created by this command:
 
 .. _-Jk:
 
-Eckert IV and VI projection (**-Jk** **-JK**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Eckert IV and VI (**-Jk** **-JK**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:1258]:
```suggestion
Eckert IV and VI projection (**-Jk** **-JK**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/rst/source/reference/map-projections.rst [hunk 4]
```diff
@@ -1219,8 +1219,8 @@ Centered on Greenwich, the example in Figure :ref:`Winkel Tripel projection <GMT
 
 .. _-Jn:
 
-Robinson projection (**-Jn** **-JN**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Robinson (**-Jn** **-JN**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:1223]:
```suggestion
Robinson projection (**-Jn** **-JN**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/rst/source/reference/map-projections.rst [hunk 5]
```diff
@@ -1177,8 +1177,8 @@ An example centered on Greenwich can be generated thus:
 
 .. _-Jr:
 
-Winkel Tripel projection (**-Jr** **-JR**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Winkel Tripel (**-Jr** **-JR**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:1181]:
```suggestion
Winkel Tripel projection (**-Jr** **-JR**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/rst/source/reference/map-projections.rst [hunk 6]
```diff
@@ -1143,8 +1143,8 @@ A view of the Pacific ocean using the Dateline as central meridian can be genera
 
 .. _-Jw:
 
-Mollweide projection (**-Jw** **-JW**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Mollweide (**-Jw** **-JW**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:1147]:
```suggestion
Mollweide projection (**-Jw** **-JW**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/rst/source/reference/map-projections.rst [hunk 7]
```diff
@@ -1110,8 +1110,8 @@ range of the plot, specified by the (**-R**) option.
 
 .. _-Jh:
 
-Hammer projection (**-Jh** **-JH**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Hammer (**-Jh** **-JH**)
+~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:1114]:
```suggestion
Hammer projection (**-Jh** **-JH**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/rst/source/reference/map-projections.rst [hunk 8]
```diff
@@ -730,8 +730,8 @@ formulae instead.
 
 .. _-Jo:
 
-Oblique Mercator projection (**-Jo** **-JO**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Oblique Mercator (**-Jo** **-JO**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:734]:
```suggestion
Oblique Mercator projection (**-Jo** **-JO**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/rst/source/reference/map-projections.rst [hunk 9]
```diff
@@ -685,8 +685,8 @@ latitudes in the regular Mercator projection and must therefore be less than 90
 
 .. _-Ju:
 
-Universal Transverse Mercator (UTM) projection (**-Ju** **-JU**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Universal Transverse Mercator (UTM) (**-Ju** **-JU**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:689]:
```suggestion
Universal Transverse Mercator (UTM) projection (**-Ju** **-JU**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/rst/source/reference/map-projections.rst [hunk 10]
```diff
@@ -627,8 +627,8 @@ example, specify the region with **-R**-180/180/-70/70 to obtain a map centered
 
 .. _-Jt:
 
-Transverse Mercator projection (**-Jt** **-JT**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Transverse Mercator (**-Jt** **-JT**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:631]:
```suggestion
Transverse Mercator projection (**-Jt** **-JT**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/rst/source/reference/map-projections.rst [hunk 11]
```diff
@@ -577,8 +577,8 @@ Each have a different way of spacing the meridians and parallels to obtain certa
 
 .. _-Jm:
 
-Mercator projection (**-Jm** **-JM**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Mercator (**-Jm** **-JM**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:581]:
```suggestion
Mercator projection (**-Jm** **-JM**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by joa-quim** [comment]
I'm fine with this.

**Comment by seisman** [comment]
> Even the `projection` word should should stay in the headers.

@yvonnefroehlich Please make sure that all headers in `doc/rst/source/reference/map-projections.rst` end with `projection`.

**Comment by yvonnefroehlich** [comment]
> > Even the `projection` word should should stay in the headers.
> 
> @yvonnefroehlich Please make sure that all headers in `doc/rst/source/reference/map-projections.rst` end with `projection`.

I apologise for the delay with this PR, I will try to continue here at the end of the week (or the weekend). Thanks a lot for all yor comments and suggestions so far 🙂!

**Comment by yvonnefroehlich** [comment]
> > Even the `projection` word should should stay in the headers.
> 
> @yvonnefroehlich Please make sure that all headers in `doc/rst/source/reference/map-projections.rst` end with `projection`.

Done in the commits  https://github.com/GenericMappingTools/gmt/pull/8567/commits/8e9d524059b1c49b94839552c8c45bd42f8f1c36 (`map-projections_.rst`) and https://github.com/GenericMappingTools/gmt/pull/8567/commits/fd04088154a3c70cfb2ff482f58e4e560f87aa41 (`proj-codes_.rst`).

**Comment by joa-quim** [comment]
Please, no **not remove** the `equal area` from headers.

File: doc/rst/source/proj-codes.rst_ [hunk 12]
```diff
@@ -17,99 +17,99 @@
    * -
      - **-J** (*scale*\|\ *WIDTH*)
      -
-   * - :ref:`Lambert azimuthal equal area <-Ja>`
+   * - :ref:`Lambert azimuthal projection <-Ja>`
```
Comment (yvonnefroehlich) [inline doc/rst/source/proj-codes.rst_:20]:
```suggestion
   * - :ref:`Lambert azimuthal equal-area projection <-Ja>`
```

File: doc/rst/source/proj-codes.rst_ [hunk 13]
```diff
@@ -17,99 +17,99 @@
...
      - **-Ja**\|\ **A**
      - |lon0|/|lat0|\ [/\ *horizon*]/\ *scale*\|\ *width*
-   * - :ref:`Albers conic equal area <-Jb>`
+   * - :ref:`Albers conic projection <-Jb>`
```
Comment (yvonnefroehlich) [inline doc/rst/source/proj-codes.rst_:23]:
```suggestion
   * - :ref:`Albers conic equal-area projection <-Jb>`
```

File: doc/rst/source/proj-codes.rst_ [hunk 14]
```diff
@@ -17,99 +17,99 @@
...
-   * - :ref:`Azimuthal orthographic <-Jg>`
+   * - :ref:`Azimuthal orthographic projection <-Jg>`
      - **-Jg**\|\ **G**
      - |lon0|/|lat0|\ [/\ *horizon*]/\ *scale*\|\ *width*
-   * - :ref:`General perspective <-Jg_pers>`
+   * - :ref:`General perspective projection <-Jg_pers>`
      - **-Jg**\|\ **G**
      - |lon0|/|lat0|\ */*\ *scale*\|\ *width*\ [**+a**\ *azimuth*][**+t**\ *tilt*][**+v**\ *vwidth/vheight*][**+w**\ *twist*][**+z**\ *altitude*\ [**r**\|\ **R**]\|\ **g**]
-   * - :ref:`Hammer equal area <-Jh>`
+   * - :ref:`Hammer projection <-Jh>`
```
Comment (yvonnefroehlich) [inline doc/rst/source/proj-codes.rst_:47]:
```suggestion
   * - :ref:`Hammer equal-area projection <-Jh>`
```

File: doc/rst/source/proj-codes.rst_ [hunk 15]
```diff
@@ -17,99 +17,99 @@
...
      - **-Jh**\|\ **H**
      - [|lon0|/]\ *scale*\|\ *width*
-   * - :ref:`Sinusoidal equal area <-Ji>`
+   * - :ref:`Sinusoidal projection <-Ji>`
```
Comment (yvonnefroehlich) [inline doc/rst/source/proj-codes.rst_:50]:
```suggestion
   * - :ref:`Sinusoidal equal-area projection <-Ji>`
```

File: doc/rst/source/proj-codes.rst_ [hunk 16]
```diff
@@ -17,99 +17,99 @@
...
      - **-Ji**\|\ **I**
      - [|lon0|/]\ *scale*\|\ *width*
-   * - :ref:`Miller cylindrical <-Jj>`
+   * - :ref:`Miller projection <-Jj>`
      - **-Jj**\|\ **J**
      - [|lon0|/]\ *scale*\|\ *width*
-   * - :ref:`Eckert IV equal area <-Jk>`
+   * - :ref:`Eckert IV projection <-Jk>`
```
Comment (yvonnefroehlich) [inline doc/rst/source/proj-codes.rst_:56]:
```suggestion
   * - :ref:`Eckert IV equal-area projection <-Jk>`
```

File: doc/rst/source/proj-codes.rst_ [hunk 17]
```diff
@@ -17,99 +17,99 @@
...
      - **-Jkf**\|\ **Kf**
      - [|lon0|/]\ *scale*\|\ *width*
-   * - :ref:`Eckert VI equal area <-Jk>`
+   * - :ref:`Eckert VI projection <-Jk>`
```
Comment (yvonnefroehlich) [inline doc/rst/source/proj-codes.rst_:59]:
```suggestion
   * - :ref:`Eckert VI equal-area projection <-Jk>`
```

File: doc/rst/source/proj-codes.rst_ [hunk 18]
```diff
@@ -17,99 +17,99 @@
...
+   * - :ref:`Mollweide projection <-Jw>`
      - **-Jw**\|\ **W**
      - [|lon0|/]\ *scale*\|\ *width*
    * - :ref:`Linear <-Jx_linear>`, :ref:`logarithmic <-Jx_log>`,
        :ref:`power <-Jx_power>`, and :ref:`time <-Jx_time>`
      - **-Jx**\|\ **X**
      - *xscale*\|\ *width*\ [**l**\|\ **p**\ *power*\|\ **T**\|\ **t**]\
        [/\ *yscale*\|\ *height*\ [**l**\|\ **p**\ *power*\|\ **T**\|\ **t**]][**d**]
-   * - :ref:`Cylindrical equal area <-Jy>`
+   * - :ref:`Cylindrical projection <-Jy>`
```
Comment (yvonnefroehlich) [inline doc/rst/source/proj-codes.rst_:113]:
```suggestion
   * - :ref:`Cylindrical equal-area projection <-Jy>`
```

File: doc/rst/source/reference/map-projections.rst [hunk 19]
```diff
@@ -49,8 +49,8 @@ Conic projections
 
 .. _-Jb:
 
-Albers conic equal-area projection (**-Jb** **-JB**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Albers conic projection (**-Jb** **-JB**)
```
Comment (yvonnefroehlich) [inline doc/rst/source/reference/map-projections.rst:52]:
```suggestion
Albers conic equal-area projection (**-Jb** **-JB**)
```

File: doc/rst/source/reference/map-projections.rst [hunk 20]
```diff
@@ -218,7 +218,7 @@ Azimuthal projections
 
 .. _-Ja:
 
-Lambert Azimuthal Equal-Area (**-Ja** **-JA**)
+Lambert azimuthal projection (**-Ja** **-JA**)
```
Comment (yvonnefroehlich) [inline doc/rst/source/reference/map-projections.rst:221]:
```suggestion
Lambert azimuthal equal-area projection (**-Ja** **-JA**)
```

File: doc/rst/source/reference/map-projections.rst [hunk 21]
```diff
@@ -308,8 +308,8 @@ stereonet can be obtained by using the stereographic projection
 
 .. _-Js:
 
-Stereographic Equal-Angle (**-Js** **-JS**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Stereographic projection (**-Js** **-JS**)
```
Comment (yvonnefroehlich) [inline doc/rst/source/reference/map-projections.rst:311]:
```suggestion
Stereographic equal-angle projection (**-Js** **-JS**)
```

File: doc/rst/source/reference/map-projections.rst [hunk 22]
```diff
@@ -941,8 +941,8 @@ A world map centered on the dateline using this projection can be obtained as fo
 
 .. _-Jy:
 
-Cylindrical equal-area projections (**-Jy** **-JY**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Cylindrical projection (**-Jy** **-JY**)
```
Comment (yvonnefroehlich) [inline doc/rst/source/reference/map-projections.rst:944]:
```suggestion
Cylindrical equal-area projection (**-Jy** **-JY**)
```

File: doc/rst/source/reference/map-projections.rst [hunk 23]
```diff
@@ -1111,7 +1111,7 @@ range of the plot, specified by the (**-R**) option.
 .. _-Jh:
 
 Hammer projection (**-Jh** **-JH**)
```
Comment (yvonnefroehlich) [inline doc/rst/source/reference/map-projections.rst:1113]:
```suggestion
Hammer equal-area projection (**-Jh** **-JH**)
```

**Comment by yvonnefroehlich** [comment]
> Please, no **not remove** the `equal area` from headers.

Done in commit https://github.com/GenericMappingTools/gmt/pull/8567/commits/40930b40f1a233846e11ad4001c4059720b290a9.

Hm. At the moment I am a bit lost regarding how I should actually change the projection names to make them more consistent /: .

**Comment by joa-quim** [comment]
Thanks. The `equal area` is an information that the projection is using an authalic surface and is important to say in the summary table. On the projections description that information is stated more explicitly (AFAIK). 

> change the projection names to make them more consistent 

Consistent with what? As I mentioned before, in my view, a table summary does not need to be exactly equal the headers in the projections page. But I understand you and confess to be also confused by now. IMO, things like they were served us perfectly during > 30 years (**were written by the author**) and didn't need any particular change.

File: doc/rst/source/reference/map-projections.rst [hunk 24]
```diff
@@ -1110,8 +1110,8 @@ range of the plot, specified by the (**-R**) option.
 
 .. _-Jh:
 
-Hammer projection (**-Jh** **-JH**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Hammer equal-area projection (**-Jh** **-JH**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:1114]:
```suggestion
Hammer projection (**-Jh** **-JH**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/rst/source/reference/map-projections.rst [hunk 25]
```diff
@@ -1002,8 +1002,8 @@ As one can see there is considerable distortion at high latitudes since the pole
 
 .. _-Jj:
 
-Miller Cylindrical projection (**-Jj** **-JJ**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Miller projection (**-Jj** **-JJ**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:1006]:
```suggestion
Miller cylindrical projection (**-Jj** **-JJ**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/rst/source/reference/map-projections.rst [hunk 26]
```diff
@@ -887,8 +887,8 @@ As with the previous projections, the user can choose between a rectangular boun
 
 .. _-Jq:
 
-Cylindrical equidistant projection (**-Jq** **-JQ**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Equidistant cylindrical projection (**-Jq** **-JQ**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:891]:
```suggestion
Cylindrical equidistant projection (**-Jq** **-JQ**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/rst/source/reference/map-projections.rst [hunk 27]
```diff
@@ -846,8 +846,8 @@ to align the oblique Equator with the vertical, positive *y*-axis instead.  This
 
 .. _-Jc:
 
-Cassini cylindrical projection (**-Jc** **-JC**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Cassini projection (**-Jc** **-JC**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:850]:
```suggestion
Cassini cylindrical projection (**-Jc** **-JC**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/rst/source/reference/map-projections.rst [hunk 28]
```diff
@@ -528,8 +528,8 @@ perimeter:
 
 .. _-Jf:
 
-Gnomonic projection (**-Jf** **-JF**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Azimuthal gnomonic projection (**-Jf** **-JF**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:532]:
```suggestion
Gnomonic projection (**-Jf** **-JF**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/rst/source/reference/map-projections.rst [hunk 29]
```diff
@@ -450,8 +450,8 @@ following :doc:`/coast` command (*lon0*\ =4; *lat0*\ =52; *altitude*\ =230 km; *
 
 .. _-Jg:
 
-Orthographic projection (**-Jg** **-JG**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Azimuthal orthographic projection (**-Jg** **-JG**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:454]:
```suggestion
Orthographic projection (**-Jg** **-JG**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/rst/source/reference/map-projections.rst [hunk 30]
```diff
@@ -396,12 +396,12 @@ hemispheric maps. Our example shows Australia using a projection pole at
...
 By choosing 0/0 as the pole, we obtain the conformal stereonet presented
-next to its equal-area cousin in the Section `Lambert Azimuthal Equal-Area (-Ja -JA)`_ on the Lambert azimuthal equal-area projection (Figure :ref:`Stereonets <GMT_stereonets>`).
+next to its equal-area cousin in the Section `Lambert azimuthal (-Ja -JA)`_ on the Lambert azimuthal projection (Figure :ref:`Stereonets <GMT_stereonets>`).
 
 .. _-Jg_pers:
 
-Perspective projection (**-Jg** **-JG**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+General perspective projection (**-Jg** **-JG**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:404]:
```suggestion
Perspective projection (**-Jg** **-JG**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/rst/source/reference/map-projections.rst [hunk 31]
```diff
@@ -308,8 +308,8 @@ stereonet can be obtained by using the stereographic projection
 
 .. _-Js:
 
-Stereographic Equal-Angle (**-Js** **-JS**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Stereographic equal-angle projection (**-Js** **-JS**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:312]:
```suggestion
Stereographic equal-angle projection (**-Js** **-JS**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/rst/source/reference/map-projections.rst [hunk 32]
```diff
@@ -218,7 +218,7 @@ Azimuthal projections
 
 .. _-Ja:
 
-Lambert Azimuthal Equal-Area (**-Ja** **-JA**)
+Lambert azimuthal equal-area projection (**-Ja** **-JA**)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:222]:
```suggestion
Lambert azimuthal equal-area projection (**-Ja** **-JA**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: doc/rst/source/reference/map-projections.rst [hunk 33]
```diff
@@ -396,12 +396,12 @@ hemispheric maps. Our example shows Australia using a projection pole at
 
 
 By choosing 0/0 as the pole, we obtain the conformal stereonet presented
-next to its equal-area cousin in the Section `Lambert Azimuthal Equal-Area (-Ja -JA)`_ on the Lambert azimuthal equal-area projection (Figure :ref:`Stereonets <GMT_stereonets>`).
+next to its equal-area cousin in the Section `Lambert azimuthal (-Ja -JA)`_ on the Lambert azimuthal projection (Figure :ref:`Stereonets <GMT_stereonets>`).
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:399]:
```suggestion
next to its equal-area cousin in the Section `Lambert azimuthal equal-area projection (-Ja -JA)`_ on the Lambert azimuthal projection (Figure :ref:`Stereonets <GMT_stereonets>`).
```

File: doc/rst/source/reference/map-projections.rst [hunk 34]
```diff
@@ -50,7 +50,7 @@ Conic projections
 .. _-Jb:
 
 Albers conic equal-area projection (**-Jb** **-JB**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (seisman) [inline doc/rst/source/reference/map-projections.rst:53]:
```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Review by seisman — COMMENTED** [review]
I guess we should keep the projection names unchanged. The suggestions below revert most changes.

File: doc/rst/source/reference/map-projections.rst (hunk 24 above)
Comment (yvonnefroehlich) [reply doc/rst/source/reference/map-projections.rst]: For this change please see commit https://github.com/GenericMappingTools/gmt/pull/8567/commits/dcc4be23c7df26b645927bdc70a928ac03e7f153.

File: doc/rst/source/reference/map-projections.rst (hunk 31 above)
Comment (yvonnefroehlich) [reply doc/rst/source/reference/map-projections.rst]: The lengths of the underlines are adjusted in commit https://github.com/GenericMappingTools/gmt/pull/8567/commits/2a26c0a431bda505bc1ce1e97096ae0ac86ad5cf.

**Review by joa-quim — APPROVED** [review]

**Pull request merged**